scripts/validate.sh

```diff
@@ -126,6 +126,25 @@ function validate_project_commits() {
     done
 }
 
+function validate_no_pin_prs() {
+    local head="$1"
+    local projects=("${@:2}")
+    local pin_prs
+
+    for project in "${projects[@]}"; do
+        if ! pin_prs="$(gh_api "pulls?base=${release['branch']:-devel}&head=${ORG}:${head}&state=open" | jq -r ".[].url")"; then
+            printerr "Failed to list pull requests for ${project}."
+            return 1
+        fi
+
+        if [[ -n "${pin_prs}" ]]; then
+            printerr "Found open ${head@Q} pull requests on ${project}, make sure they're merged before proceeding"
```

Comment:
If anyone else is wondering about `@Q`:

```
${varname@Q} returns a single-Quoted string with any special characters (such as \n, \t, etc.) escaped.
```

https://stackoverflow.com/a/41940626/1011749